### PR TITLE
Added Indonesia translation

### PR DESCRIPTION
**Pull request by muhmudrik**

### CODE REVIEW

File: src/translations/id.lua [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+		[040] = { "Penyihir", "Penyihir Wanita" },
+		[050] = "Pengendali Kenyataan",
+		[060] = { "Penguasa Mantra", "Nyonya Mantra" },
+		[070] = "Pemanggil Perdukunan",
+		[080] = "Penunggang Kuda Wabah",
+		[090] = "Penunggang Kuda Kelaparan",
+		[100] = "Penunggang Kuda Perang",
+		[110] = "Penunggang Kuda Kematian",
+		[120] = "Kekosongan"
+	},
```
Comment (Lautenschlager-id) [inline src/translations/id.lua:76]: Can you confirm you have done {'male version', 'female version'} for all titles that could have both genders?

Comment (muhmudrik) [reply src/translations/id.lua:76]: For some occupations in Indonesian, it becomes gender-neutral. So I think I've done it all.

**Review by Lautenschlager-id — APPROVED** [review]
Can you change the branch to :develop instead of :master, please?

**Comment by muhmudrik** [comment]
Done!

**Comment by Lautenschlager-id** [comment]
Please, what is your nickname ingame?

**Comment by muhmudrik** [comment]
Rokamudrik#0000

**Pull request merged**